final/app/src/main/java/com/example/makeitso/model/service/impl/StorageServiceImpl.kt

```diff
@@ -46,8 +56,9 @@ constructor(private val firestore: FirebaseFirestore, private val auth: AccountS
 
   override suspend fun save(task: Task): String =
     trace(SAVE_TASK_TRACE) {
-      val taskWithUserId = task.copy(userId = auth.currentUserId)
-      firestore.collection(TASK_COLLECTION).add(taskWithUserId).await().id
+      val instant = ISO_INSTANT.format(Instant.now())
```

Comment:
Do we have a reason to use the local time rather than the [server time](https://firebase.google.com/docs/firestore/manage-data/add-data#server_timestamp)?

Reply:
Changed to use server time!